autogen/main/versions.tf.tmpl

```diff
@@ -24,11 +24,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 5.40.0, < 7"
+      version = ">= 5.44.0, < 7, !=6.0.0, !=6.0.1, !=6.1.0"
```

Comment:
Let me know if there's a better way to specify > 5.44.0 or > 6.2.0.

Reply:
Unfortunately looking at the underlying library it appears the constraints are AND, there isn't a way to group/OR them.

https://github.com/hashicorp/go-version